securedrop/sdconfig.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+
+import config as _config
+
+import typing
+# https://www.python.org/dev/peps/pep-0484/#runtime-or-type-checking
+if typing.TYPE_CHECKING:
+    # flake8 can not understand type annotation yet.
+    # That is why all type annotation realted import
+    # statements has to be marked as noqa.
+    # http://flake8.pycqa.org/en/latest/user/error-codes.html?highlight=f401
+    from typing import List, Dict  # noqa: F401
+
+
+class SDConfig(object):
+    def __init__(self):
+        # type: () -> None
+        self.FlaskConfig = _config.FlaskConfig
```

Comment:
I think we should use the `hasattr` guard for every single member of the `config.py` module. This covers us to be dead sure that we're not going to raise exceptions at runtime on a prod instance.